### ISSUE TITLE
Trade button for timeless jewel update

### ISSUE THREAD

**Issue by KeshHere** [Lailloken/Lailloken-UI#74]
https://imgur.com/a/iIKNVHn
I have it ready on the fork [here](https://github.com/KeshHere/Lailloken-UI) so if enough people are interested and if Lailloken agrees this can be added in this main version.

**Comment by KeshHere** [comment]
Here is the code
```
Gui, legion_window: Add, Button, % "x180 y70 BackgroundTrans Center gtrade_open w80 h25", % "Trade"

trade_open:
	Gui, legion_treemap: Destroy
	hwnd_legion_treemap := ""
	Gui, legion_window: Destroy
	hwnd_legion_treemap := ""
	if (legion_type_parse = "Elegant Hubris") {
		tradelink := "{%22query%22:{%22status%22:{%22option%22:%22online%22},%22stats%22:[{%22type%22:%22count%22,%22filters%22:[{%22id%22:%22explicit.pseudo_timeless_jewel_caspiro%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_cadiro%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_victario%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false}],%22value%22:{%22min%22:1}}]},%22sort%22:{%22price%22:%22asc%22}}"
	}
	if (legion_type_parse = "Brutal Restraint"){
		tradelink := "{%22query%22:{%22status%22:{%22option%22:%22online%22},%22stats%22:[{%22type%22:%22count%22,%22filters%22:[{%22id%22:%22explicit.pseudo_timeless_jewel_asenath%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_balbala%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_nasima%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false}],%22value%22:{%22min%22:1}}]},%22sort%22:{%22price%22:%22asc%22}}"
	}
	if (legion_type_parse = "Lethal Pride"){
		tradelink :="{%22query%22:{%22status%22:{%22option%22:%22online%22},%22stats%22:[{%22type%22:%22count%22,%22filters%22:[{%22id%22:%22explicit.pseudo_timeless_jewel_akoya%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_kaom%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_rakiata%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false}],%22value%22:{%22min%22:1}}]},%22sort%22:{%22price%22:%22asc%22}}"
	}
	if (legion_type_parse = "Militant Faith"){
		tradelink :="{%22query%22:{%22status%22:{%22option%22:%22online%22},%22stats%22:[{%22type%22:%22count%22,%22filters%22:[{%22id%22:%22explicit.pseudo_timeless_jewel_avarius%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_dominus%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_maxarius%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false}],%22value%22:{%22min%22:1}}]},%22sort%22:{%22price%22:%22asc%22}}"
	}
	if (legion_type_parse = "Glorious Vanity"){
		tradelink := "{%22query%22:{%22status%22:{%22option%22:%22online%22},%22stats%22:[{%22type%22:%22count%22,%22filters%22:[{%22id%22:%22explicit.pseudo_timeless_jewel_ahuana%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_doryani%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false},{%22id%22:%22explicit.pseudo_timeless_jewel_xibaqua%22,%22value%22:{%22min%22:" . legion_seed_parse . ",%22max%22:" . legion_seed_parse . "},%22disabled%22:false}],%22value%22:{%22min%22:1}}]},%22sort%22:{%22price%22:%22asc%22}}"
	}

	
	Run, https://www.pathofexile.com/trade/search/Sentinel?q=%tradelink%
	


Return
```

**Comment by Lailloken** [comment]
Implemented in v1.26.2. Thanks for the suggestion and the query-code.
